include/kllvm/codegen/ProofEvent.h

```diff
@@ -104,7 +104,7 @@ class proof_event {
       llvm::BasicBlock *current_block);
 
   [[nodiscard]] llvm::BasicBlock *rewrite_event_pre(
-      sptr<kore_axiom_declaration> axiom, uint64_t arity,
+      sptr<kore_axiom_declaration> &axiom, uint64_t arity,
```

Comment:
Why are these changed to be non-const references? In general I'm suspicious of code that needs a mutable reference to a smart pointer.

Reply:
I was fixing a clang-tidy issue, but I forgot to make them `const`, sorry!